README.md

```diff
@@ -40,15 +43,42 @@ It’s a Rails app that:
     $ bundle install --without production
     ```
 
-7. Set up your database. The simple way to do this is:
+7. Set up your database
 
+    Make a local copy of the database configuration template file:
+    
     ```sh
-    $ bundle exec rake db:setup
+    cp config/database.yml.example config/database.yml
+    ```
+
+    First create users, databases and configure extensions:
+
+    ```sh
+    sudo -u postgres psql -c "CREATE USER wmdb_dev WITH PASSWORD 'wmdb_dev';"
+    sudo -u postgres createdb -O wmdb_dev wmdb_dev -E utf-8
```

Comment:
`rake db:create` (or `rake db:setup`) should use the username and password specified in `DATABASE_URL` or `database.yml`. I don’t think this line is necessary.